concrete/bootstrap/configure.php

```diff
@@ -84,6 +84,7 @@
  * Directory names
  * ----------------------------------------------------------------------------
  */
+const DIRNAME_SRC = 'src';
```

Comment:
We already have `DIRNAME_CLASSES`: what about using in instead of adding this new constant?